clients/clearly-defined/src/main/kotlin/Curation.kt

```diff
@@ -17,15 +17,12 @@
  * License-Filename: LICENSE
  */
 
-@file:UseSerializers(FileSerializer::class, URISerializer::class)
-
 package org.ossreviewtoolkit.clients.clearlydefined
 
-import java.io.File
-import java.net.URI
+import io.ks3.java.`typealias`.FileAsString
```

Comment:
That's a very inconvenient choice for a package name.

Reply:
I've [reported](https://github.com/Kantis/ks3/issues/113) it.